SIMS/SIMS/Product.cs

```diff
@@ -0,0 +1,15 @@
+namespace SIMS;
+
+public class Product
```

Comment:
Generally speaking, the vast majority of cases in real world problems, each entity should has its own `Identity`, a unique `Id` and it's used for manipulating that Id, what about having an Id for `product`? it could be "Guid" or "int", that would be more realistic. Also that would reflect on the update and delete methods for example, the Ids should be used instead of names, though you can keep your constraint of name uniqueness if you want.